src/Utils/utils.ts

```diff
@@ -240,6 +240,7 @@ export const parsePhoneNumber = (phoneNumber: string, countryCode?: string) => {
   if (phoneNumber === "+91") return "";
   const phoneCodes: Record<string, CountryData> = phoneCodesJson;
   let parsedNumber = phoneNumber.replace(/[-+() ]/g, "");
+  if (parsedNumber.length < 12) return "";
```

Comment:
this could make things problematic for toll free numbers or landlines

Reply:
@khavinshankar will be migrating the current phone number field to the new UI after this PR gets merged

Reply:
<img width="607" alt="image" src="https://github.com/user-attachments/assets/d28e44cd-11c0-4a8d-a644-01f69920e925" />
Lets use the phonenumber field instead of the custom one.